resources/views/includes/sidebar.blade.php

```diff
@@ -9,7 +9,8 @@
                 </a>
             </li>
             <li>
-                <a href="">
+                <!-- redirects to profile page based on username -->
+                <a href="../profile/{{ Auth::user()->username }}">
```

Comment:
Did some testing and I seem to get a url of 'http://split.app/profile/profile/dan' when using "./" after clicking on User Profile from the profile page.

However, I get the correct url of ''http://split.app/profile/dan' when using "../" using the same workflow.